<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddQuestionWindow</class>
 <widget class="QDialog" name="AddQuestionWindow">
  <property name="windowTitle">
   <string>Add Question</string>
  </property>

  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">

     <item row="0" column="0">
      <widget class="QLabel" name="labelCourse">
       <property name="text"><string>Course:</string></property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtCourse"/>
     </item>

     <item row="1" column="0">
      <widget class="QLabel" name="labelTopic">
       <property name="text"><string>Topic:</string></property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtTopic"/>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="labelQuestion">
       <property name="text"><string>Question:</string></property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="txtQuestion"/>
     </item>

     <item row="3" column="0">
      <widget class="QLabel" name="labelDifficulty">
       <property name="text"><string>Difficulty:</string></property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboDifficulty"/>
     </item>

     <item row="4" column="0">
      <widget class="QLabel" name="labelCLO">
       <property name="text"><string>CLO:</string></property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtCLO"/>
     </item>

     <item row="5" column="0">
      <widget class="QLabel" name="labelMarks">
       <property name="text"><string>Marks:</string></property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spinMarks">
       <property name="minimum"><number>1</number></property>
       <property name="maximum"><number>100</number></property>
      </widget>
     </item>

    </layout>
   </item>

   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text"><string>Save</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text"><string>Cancel</string></property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
